apps/judicial-system/web/src/routes/Shared/Cases/PrisonCases.tsx

```diff
@@ -313,8 +319,20 @@ export const PrisonCases: FC = () => {
                 )}
               />
               <Box marginBottom={[5, 5, 12]}>
-                {loading || !user || indictmentCases.length > 0
-                  ? renderIndictmentTable(indictmentCases)
+                {loading || !user || unregisteredCases.length > 0
+                  ? renderIndictmentTable(unregisteredCases)
+                  : renderAlertMessage()}
+              </Box>
+
+              <SectionHeading
+                title={formatMessage(
+                  m.activeRequests.prisonStaffUsers
+                    .prisonAdminIndictmentCaseTitleRegisteredCases,
+                )}
+              />
+              <Box marginBottom={[5, 5, 12]}>
+                {loading || !user || registeredCases.length > 0
+                  ? renderIndictmentTable(registeredCases)
```

Comment:
Adding a new case list here is fine, but it will be irrelevant once we implement the new case table scheme.